### ISSUE TITLE
Wantlist artifact not generated?

### ISSUE THREAD

**Issue by Thwar** [info-sharing-environment/NIEM-Modeling-Tool#9]
Wantlist is a required artifact for IEPD development. The plugin does not generate a complete IEPD without the wantlist. Does this have to be done manually?

**Comment by aowen** [comment]
Thomas, I don’t believe the concept of a “wantlist” is in scope of the NIEM Profile 2.0.  If my memory serves me, this is because the wantlist is used for the subset schema generation tool and the thinking is if you’re using a NIEM UML tool you don’t need the SSGT.

-Andrew
916.215.3933

**Comment by CoryCasanave** [comment]
I would agree, the "want list" is an artifact to support the legacy XML tools and should not be a "required artifact".

**Comment by Thwar** [comment]
Just read the Model Package Description Specification 3.0.1 (27 April 2015). Wantlist is no longer a required artifact. 

I was referencing to this document before: http://reference.niem.gov/niem/guidance/iepd-requirements/2.1/iepd-requirements-2.1.pdf where Wantlist is a required artifact.
